spec-main/api-web-contents-spec.ts

```diff
@@ -1521,21 +1521,37 @@ describe('webContents module', () => {
 
   describe('setBackgroundThrottling()', () => {
     afterEach(closeAllWindows);
-    it('does not crash when allowing', () => {
+    it('does not crash when allowing via getter', () => {
       const w = new BrowserWindow({ show: false });
       w.webContents.setBackgroundThrottling(true);
+      expect(w.webContents.getBackgroundThrottling()).to.equal(true);
```

Comment:
A test that checks that there's no crash shouldn't fail if some expectation is not met.
Either change the test description or remove the `expect()` call.